xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -3764,6 +3776,13 @@ bool CVideoPlayer::CloseStream(CCurrentStream& current, bool bWaitForBuffers)
   if(bWaitForBuffers)
     SetCaching(CACHESTATE_DONE);
 
+  if (STREAM_SOURCE_MASK(current.source) == STREAM_SOURCE_DEMUX)
+  {
+    if (!m_pDemuxer)
+      return false;
```

Comment:
From reading the code: There is an OpenStream() some lines above, it calls "EnableStream(true), then CloseStream() should call EnableStream(false) or do I something wrong.

Situation is, that the stream is not disabled when seek is called on AudioChange, this leads to the fact that I have to seek 3 streams: video / audio old and audio new.

Pls. let me know at what place it has to be called